project/plugins.sbt

```diff
@@ -23,3 +23,9 @@ libraryDependencies += "org.ow2.asm"  % "asm" % "5.0.3"
 libraryDependencies += "org.ow2.asm"  % "asm-commons" % "5.0.3"
 
 libraryDependencies += "org.antlr" % "antlr" % "3.5.2"
+
+
+// TODO I am not sure we want such a dep.
+resolvers += "simplytyped" at "http://simplytyped.github.io/repo/releases"
```

Comment:
This is for the ANTLR4 SBT plugin. I am just curious if we want to depend on a github repo in our build.

This will remain in the build even after we drop ANTLR3.